src/Administration/Resources/app/administration/src/module/sw-category/view/sw-category-detail-products/index.js

```diff
@@ -213,6 +213,23 @@ export default {
             }
         },
 
+        getItemName(product) {
+            const name = product.name ? product.name : product.translated.name;
+            if(name){
```

Comment:
```suggestion
            if (name) {
```
Will fail with our linters otherwise